code/modules/organs/blood.dm

```diff
@@ -232,6 +239,11 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 	. = ..()
 	vessel.remove_reagent(BLOOD,amount) // Removes blood if human
 
+/mob/living/carbon/monkey/take_blood(obj/item/weapon/reagent_containers/container, var/amount)
+	if(!isDead())
```

Comment:
Since monkeys dont have a blood container I made it so you can take blood from a dead monkey, and they will slowly begin to die from oxyloss based on how much blood you take from them.